container/layer.bzl

```diff
@@ -284,6 +284,7 @@ _layer_attrs = dicts.add({
         mandatory = False,
     ),
     "portable_mtime": attr.bool(default = False),
+    "preserve_tar_mtimes": attr.bool(default = False),
```

Comment:
I'm not seeing this flag passed on to the build_tar.py binary called [here](https://github.com/bazelbuild/rules_docker/blob/8a924f69ebdc1514a230bbf8b08c0edefbb4af9f/container/layer.bzl#L145). Are you sure this is working as intended? Might make sense to add a test on a container_image rule for this as well. The `container_test` rule allows you to check file timestamps I think

Reply:
You're right, thanks for catching that - I missed the lines that copy over that argument while I was copying my changes into my forked git repo. I'll look into adding those tests now.